proto/desmos/profiles/v1beta1/msgs.proto

```diff
@@ -207,10 +219,67 @@ message BlockUserResponse {}
 
 // MsgUnblockUser represents a message to unblock a previously blocked user.
 message MsgUnblockUser {
-  string blocker = 1 [(gogoproto.moretags) = "yaml:\"blocker\""];
-  string blocked = 2 [(gogoproto.moretags) = "yaml:\"blocked\""];
-  string subspace = 4 [(gogoproto.moretags) = "yaml:\"subspace\""];
+  string blocker = 1 [ (gogoproto.moretags) = "yaml:\"blocker\"" ];
+  string blocked = 2 [ (gogoproto.moretags) = "yaml:\"blocked\"" ];
+  string subspace = 4 [ (gogoproto.moretags) = "yaml:\"subspace\"" ];
 }
 
 // UnblockUserResponse defines the Msg/UnblockUser response type.
 message UnblockUserResponse {}
+
+// ___________________________________________________________________________________________________________________
+
+// MsgIBCCreateAccountLink represents the message to be used to create a
+// link.
+message MsgCreateIBCAccountLink {
```

Comment:
What is this used for? Can this be removed?

Reply:
I would like to use it to test the ibc packet with docker.

Reply:
IMO this should be removed. If we keep this message everyone will be able to use it, users included. For this reason, any test message that might be useful when performing automated tests should be declared as an internal class and defined inside the test classes themselves. We should not define something only because it might be useful inside tests